bagel/tests/conftest.py

```diff
@@ -26,6 +26,16 @@ def test_data():
     return Path(__file__).absolute().parent / "data"
 
 
+@pytest.fixture(scope="session")
+def data_upload_path(tmp_path_factory):
+    return Path(__file__).absolute().parent.parent.parent / "data-upload"
+
+
+@pytest.fixture(scope="session")
+def test_pheno_data(data_upload_path):
```

Comment:
I'm assuming you named this fixture test_**pheno**_data since only the purely phenotypic files are used in the tests, which is fair! That said, since this path also encompasses the `pheno-bids-output` (which we may test against in the future), we may want to give ourselves a bit more flexibility by renaming this to something like `data_upload_path` or `test_data_upload_path` for clarity.